pkg/security/auth_test.go

```diff
@@ -25,23 +25,34 @@ import (
 
 // Construct a fake tls.ConnectionState object. The spec is a semicolon
 // separated list if peer certificate specifications. Each peer certificate
-// specification is a comma separated list of names where the first name is the
+// specification can have an optional OU in parenthesis followed by
+// a comma separated list of names where the first name is the
 // CommonName and the remaining names are SubjectAlternateNames. For example,
 // "foo" creates a single peer certificate with the CommonName "foo". The spec
 // "foo,bar" creates a single peer certificate with the CommonName "foo" and a
-// single SubjectAlternateName "bar". Contrast that with "foo;bar" which
-// creates two peer certificates with the CommonNames "foo" and "bar"
-// respectively.
+// single SubjectAlternateName "bar". "(Tenants)foo,bar" creates a single
+// tenant client certificate with OU=Tenants, CN=foo and sibjectAlernativeName=bar
+// Contrast that with "foo;bar" which creates two peer certificates with the
+// CommonNames "foo" and "bar" respectively.
 func makeFakeTLSState(spec string) *tls.ConnectionState {
 	tls := &tls.ConnectionState{}
 	if spec != "" {
 		for _, peerSpec := range strings.Split(spec, ";") {
+			var ou []string
+			if peerSpec[:1] == "(" {
```

Comment:
nit: I know this is test code, but `strings.HasPrefix(peerSpec, "(")` would avoid the potential NPE if `peerSpec == ""`.

Reply:
done